### ISSUE TITLE
"Change theme" does not work properly

### ISSUE THREAD

**Issue by EvgeniyDoctor** [svenstaro/miniserve#872]
"Change theme" absolutely does not work on **iOS**. I tried in Safari and Brave, no reaction at all.

It does not work properly even on my **computer** in **Brave in mobile view**. When i click on the button, theme immediately changes:
![ChangeTheme](https://user-images.githubusercontent.com/15073200/184785660-f0d969ab-533f-4160-9e43-57cae0ec6851.gif)

On Android it works well.

- miniserve: 0.20.0 in docker
- MacOS 11.6.8:
  - Brave: Version 1.42.88 Chromium: 104.0.5112.81 (Official Build) (x86_64)]
- iOS 15.6:
  - Brave: 1.41.1
  - Safari: 15.6
- Android:
  - Brave: 1.41.100

**Comment by svenstaro** [comment]
Can you still reproduce this?

**Comment by EvgeniyDoctor** [comment]
No, now it works. But in iOS Safari this button does not do anything.

**Comment by ahti** [comment]
It works as expected for me on Safari on iOS 18.3.2. What version are you running, and do you have any content blockers/other extensions that might be interfering?
